cmd/copy.go

```diff
@@ -614,6 +614,20 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 		glcm.SetOutputFormat(common.EOutputFormat.None())
 	}
 
+	cooked.preserveSMBInfo = areBothLocationsSMBAware(cooked.fromTo)
+	// If user has explicitly specified not to copy SMB Information, set cooked.preserveSMBInfo to false
+	if !raw.preserveSMBInfo {
+		cooked.preserveSMBInfo = false
+	}
+
+	if err = validatePreserveSMBPropertyOption(cooked.preserveSMBInfo, cooked.fromTo, &cooked.forceWrite, "preserve-smb-info"); err != nil {
+		return cooked, err
+	}
+
+	if cooked.preserveSMBInfo && !raw.preserveSMBPermissions {
+		glcm.Info("preserve-smb-permissions flag is set to false. AzCopy will not copy SMB ACLs between aware resources.")
```

Comment:
Minor: consider rephrasing:
     Please note: the preserve-smb-permissions flag is set to false, thus AzCopy will not copy SMB ACLs between the source and destination.